internal/server/errors.go

```diff
@@ -78,20 +72,11 @@ func sendAPIError(c *gin.Context, err error) {
 	case errors.Is(err, context.DeadlineExceeded):
 		resp.Code = http.StatusGatewayTimeout // not ideal, but StatusRequestTimeout isn't intended for this.
 		resp.Message = "request timed out"
-
-	default:
-		log = logging.L.WithOptions(zap.AddCallerSkip(1)).Error
 	}
 
-	log("api request error",
-		zap.String("method", c.Request.Method),
-		zap.String("path", c.Request.URL.Path),
-		zap.Int32("statusCode", resp.Code),
-		zap.String("remoteAddr", c.Request.RemoteAddr),
-		zap.Error(err))
-
 	c.JSON(int(resp.Code), resp)
-	c.Abort()
+	//nolint:errcheck // AbortWithError returns the error it pushed to the context or panics of err is nil
+	c.AbortWithError(int(resp.Code), err)
```

Comment:
This means that every invalid login attempt gets logged as an error. I think it should be at the `info` level to avoid letting unauthorized outside actors flood the logs (it can be a separate change).

Reply:
That was already the case before so this isn't introducing anything new

Reply:
This did change the behaviour. Previously we used `Debug` level for most errors, `Info` for authn errors, and `Error` for 5xx errors.  With this change I think we'd have everything at info level in the request logs.

Reply:
You're right. I forgot logs levels are dependent on the error type

Reply:
Splitting it out for now. The current implementation isn't ideal so we should figure something out